africanus/gridding/wgridder/tests/test_wgridder.py

```diff
@@ -0,0 +1,403 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from numpy.testing import assert_allclose, assert_array_almost_equal
+import pytest
+from africanus.constants import c as lightspeed
+
+pmp = pytest.mark.parametrize
+
+
+def _l2error(a, b):
+    return np.sqrt(np.sum(np.abs(a-b)**2)/np.maximum(np.sum(np.abs(a)**2),
+                                                     np.sum(np.abs(b)**2)))
+
+
+def explicit_gridder(uvw, freq, ms, wgt, nxdirty, nydirty, xpixsize, ypixsize,
```

Comment:
It would be nice to jit this function to speed up tests (if the dimensions are large).

Reply:
Get the following when I try this
```
E           NotImplementedError: Failed in nopython mode pipeline (step: analyzing bytecode)
E           CALL_FUNCTION_EX with **kwargs not supported
```
I guess this is because of the indexing='ij' inside np.meshgrid. This function is only called a handful of times (only for test_gridder()) and the tests don't take long. I don't think it will really make a big difference